### PR TITLE
Add a jq version for arm64

### PR DESCRIPTION
**Pull request by fmeum**

### CODE REVIEW

File: MODULE.bazel [hunk 1]
```diff
@@ -341,6 +341,13 @@ http_file(
     urls = ["https://github.com/jqlang/jq/releases/download/jq-1.5/jq-osx-amd64"],
 )
 
+http_file(
+    name = "jq_macos_arm64",
+    executable = True,
+    integrity = "sha256-C75hnmY+DeLFUL4v4NJA0HZ5nW+KZStw+gSuqKg2Loo=",
+    urls = ["https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-macos-arm64"],
```
Comment (meteorcloudy) [inline MODULE.bazel:348]: Is it fine this isn't version 1.5? What are the differences?

Comment (fmeum) [reply MODULE.bazel:348]: I would like to keep some `jq`s at the minimum supported version (1.5) to ensure that we don't start using features available only in more recent versions. But 1.7 is the first minor release with arm64 builds.

Comment (meteorcloudy) [reply MODULE.bazel:348]: I see, I guess 1.7 should be backwards compatible, right? And with other platform staying at 1.5, we should not be able to use new features in 1.7.

Comment (meteorcloudy) [reply MODULE.bazel:348]: Can we also add for Linux arm64 and Windows arm64?

Comment (meteorcloudy) [reply MODULE.bazel:355]: Ah,, I forgot to send my comment earlier... sorry

**Review by meteorcloudy — APPROVED** [review]

**Review by meteorcloudy — APPROVED** [review]

**Comment by fmeum** [comment]
@bazel-io fork 7.3.0

File: scripts/BUILD [hunk 2]
```diff
@@ -40,12 +40,28 @@ sh_test(
...
+filegroup(
+    name = "jq_macos",
+    srcs = select({
+        "@platforms//cpu:arm64": ["@jq_macos_arm64//file"],
+        "@platforms//cpu:x86_64": ["@jq_macos_amd64//file"],
+    }),
+)
+
+filegroup(
+    name = "jq_windows",
```
Comment (meteorcloudy) [inline scripts/BUILD:52]: This should be linux instead

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**